homeassistant/components/prowl/__init__.py

```diff
@@ -1 +1,47 @@
 """The prowl component."""
+
+import logging
+
+import prowlpy
+
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
+
+from .const import DOMAIN, PLATFORMS
+from .helpers import async_verify_key
+from .notify import PLATFORM_SCHEMA, ProwlConfigEntry
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["DOMAIN", "PLATFORM_SCHEMA"]
```

Comment:
Why do we need to export `DOMAIN` and `PLATFORM_SCHEMA`?

Reply:
I could have sworn it was needed at some point, but clearly doesn't anymore. Removed.